apps/web/test/web/controllers/opportunities_controller_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule Web.OpportunitiesControllerTest do
+  use Web.ConnCase
+
+  test "GET /list", %{conn: conn} do
+    conn = get conn, "/list"
+    assert html_response(conn, 200) =~ "Extracurricular"
```

Comment:
To thoroughly test this we'll need some data.  To do that let's import the factories from the Data project using `import Data.Factory` after `use` and before our first test case declaration.

Inside our test case we should insert some opportunities and grab something relevant from them to look for in the HTML, perhaps their title?

```elixir
[%{title: first_title}, %{title: second_title}] = insert_pair(:opportunity)

assert html_response(conn, 200) =~ first_title
assert html_response(conn, 200) =~ second_title
```

Just an idea, thoughts?